### ISSUE TITLE
Autosave does not work for FTML files

### ISSUE THREAD

**Issue by zachthieme** [atom/autosave#51]
I'm using https://github.com/FoldingText/foldingtext-for-atom which uses the FTML document type.  When i navigate to a different tab it does not save for FTML files other file types save as expected.

Atom version: 1.0.19
Recently removed the Chocolatey installer version and installed via the new Squirrel-based installer

**Comment by izuzak** [comment]
Thanks for the report, @zachthieme, I was able to reproduce this. I'm thinking it's because of [this line](https://github.com/atom/autosave/blob/2bccd0baf38ed636960d020bf0b34d5ed3fcfc5b/lib/autosave.coffee#L23): `if event.target.matches('atom-text-editor:not([mini])')`. The target element for ftml files does not match that (I just checked), so the condition fails. However, autosave works correctly when the whole window loses focus since in that case the check is not performed.

I'm wondering if @nathansobo can share some thoughts on whether we could relax that condition so that autosave works for non atom-text-editor element (blame indicates that you were the last person to change this code :grin: :v:)? The pane item in question does satisfy the other requirements, listed [here](https://github.com/atom/autosave/blob/2bccd0baf38ed636960d020bf0b34d5ed3fcfc5b/lib/autosave.coffee#L34-L38), and again -- it autosaves correctly when the whole window is blurred, so I'm hopeful we can relax that.

**Comment by nathansobo** [comment]
Yes, I do think we can relax the selector based constraint and just use the criteria you listed.

**Comment by izuzak** [comment]
> Yes, I do think we can relax the selector based constraint and just use the criteria you listed.

Cool, thanks for taking a look, @nathansobo :bow:

@zachthieme With that information :point_up:, are you perhaps interested in making a pull request to modify this behavior? @jessegrosjean might be interested as well. :smile:
